.claude/CLAUDE.md

```
@@ -55,12 +55,14 @@ Use the `bin/test-rust` script to run the tests — it is the standard, supporte
 ```bash
 bin/test-rust                         # Build and run all Rust tests
 bin/test-rust test_function_name      # Run only Rust tests matching test_function_name
+bin/test-rust --timings test_name     # Report each matching test's duration
 bin/test-rust -p                      # Run the Python test suite (via pytest + maturin)
 bin/test-rust -p -k test_init         # Run Python tests matching test_init
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)